test/gtest/ucs/test_profile.cc

```diff
@@ -335,11 +335,13 @@ void test_profile::do_test(unsigned int_mode, const std::string& str_mode)
                 break;
             };
 
-            test_nesting(loc, nesting, "profile_test_func1", 0);
-            test_nesting(loc, nesting, "code", 1);
+            /* Test nesting expects SCOPE_BEGIN to have name association
+             * unlike previously where SCOPE_END had the name association */
+            test_nesting(loc, nesting, "profile_test_func1", 1);
```

Comment:
1. need to update read_profile tool as well, to take name from start and not from end
2. is it feasible to add some testing to test_jenkins.sh (profile test section) to run nvtx tool and check that things were recorded?